src/libArgParse/Alternation.hpp

```diff
@@ -102,6 +102,16 @@ class Alternation : public GrammarElement
                         //std::cout << "  Alternation"<< std::to_string(m_instanceId) << ": have possible candidate: '" << candidate->getMatchedString() << "'" << std::endl;
                         candidateList.push_back(candidate);
```

Comment:
we most probably do not want to add elements as completion candidates if we could not retrieve grammar.
maybe we should change line 92 to only handle "missingText" errors here.